src/image.cpp

```diff
@@ -278,6 +284,7 @@ namespace Exiv2 {
         return Image::byteSwap(v,bSwap);
     }
 
+    /// \todo not used internally. At least we should test it
```

Comment:
I added these comments in few places because cppcheck was detecting these functions as UNUSED ones. However they are part of the API and they should not be removed without further consideration.